cmd/ctr/utils.go

```diff
@@ -100,7 +100,7 @@ func forwardAllSignals(containers execution.ContainerServiceClient, id string) c
 			}
 			_, err := containers.Kill(gocontext.Background(), killRequest)
 			if err != nil {
-				logrus.WithError(err).Error("grpc event from kill")
+				logrus.Fatalln(err)
```

Comment:
ah nice , 
I didn't know. I did the changes , but it is too small for another pull request so maybe will combine with something else.